production/ksonnet/loki/config.libsonnet

```diff
@@ -47,6 +47,7 @@
       // A higher value will lead to a querier trying to process more requests than there are available
       // cores and will result in scheduling delays.
       concurrency: 4,
+      multi_tenant_queries_enabled: null,
```

Comment:
Q: why not just boolean?
`multi_tenant_queries_enabled: false`. 

which would also make below conditional check bit easier

Reply:
:see_no_evil:

Reply:
what's the behavior if it's not included?

Reply:
Depends on the Loki version. In older versions Loki would crash. I'd like to avoid that. If it was not included it would default to `false`, for now.

Overall I want to keep backwards compatibility.

Reply:
It raises an interesting question: are customers using mixins from the release branch? I don't know if we give any specific advice in that regard.

I think we have so many other BC-breaks in that case. Maybe we need to introduce a `version` field in our mixins and conditionally include config based on that? Or some kind of release config like we do for Grafana Cloud Logs?

Reply:
I agree. However, this would probably blow up the debate here. Also I'm thinking about our deployments. I'm afraid we would break them.